uxarray/grid/grid.py

```diff
@@ -1864,14 +1864,21 @@ def encode_as(self, grid_type: str) -> xr.Dataset:
         warn(
             "Grid.encode_as will be deprecated in a future release. Please use Grid.to_xarray instead."
         )
+        grid_type = grid_type.lower()
+        # Node boundaries/coords are not populated by default. Accessing node_lon will trigger their population.
+        # Do this before encoding to ensure that the node boundaries/coords are populated.
+        if "node_lon" not in self._ds.variables:
+            _ = self.node_lon
```

Comment:
We don't need the if statement here, since that will trigger if we simply call the `node_lon` property.

Reply:
Good observations. Fixed.